controllers/openshift_controller.go

```diff
@@ -2033,6 +2041,69 @@ func (r *KataConfigOpenShiftReconciler) addInProgressCondition() *kataconfigurat
 	return &r.kataConfig.Status.Conditions[len(r.kataConfig.Status.Conditions)-1]
 }
 
+// This is just a technical helper to all InProgress Condition mutators,
+// factoring their common preamble out into an own function.
+func (r *KataConfigOpenShiftReconciler) retrieveInProgressConditionForChange() *kataconfigurationv1.KataConfigCondition {
+	cond := r.findInProgressCondition()
+	if cond == nil {
+		cond = r.addInProgressCondition()
+	}
+
+	cond.LastTransitionTime = metav1.Now()
+
+	return cond
+}
+
+func (r *KataConfigOpenShiftReconciler) setInProgressConditionToInstalling() {
+	cond := r.retrieveInProgressConditionForChange()
+	cond.Status = corev1.ConditionTrue
+	cond.Reason = "Installing"
+	cond.Message = "Performing initial installation on cluster"
```

Comment:
To help grepability in some log blob and for consistency with the other mutators :

```suggestion
	cond.Message = "Performing initial installation of kata on cluster"
```

Reply:
Makes sense, thanks - it's fixed now.